### PR TITLE
[Hyperdrive] Run prettier, fix-up wrangler commands, types

### PR DESCRIPTION
**Pull request by rozenmd**

### CODE REVIEW

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/hyperdrive/_partials/_create-hyperdrive-config.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ export default {
 			// Test query
 			let result = await client.query({ text: 'SELECT * FROM pg_tables' });
 
+			ctx.waitUntil(client.end());
```
Comment (rozenmd) [inline content/hyperdrive/_partials/_create-hyperdrive-config.md]: @elithrar that's right, yeah?

File: content/hyperdrive/get-started.md [hunk 2]
```diff
@@ -223,6 +223,7 @@ export default {
 			// Test query
 			let result = await client.query({ text: 'SELECT * FROM pg_tables' });
 
+			ctx.waitUntil(client.end());
```
Comment (rozenmd) [inline content/hyperdrive/get-started.md]: ditto here @elithrar

File: content/hyperdrive/learning/connect-to-postgres.md [hunk 3]
```diff
@@ -114,7 +115,7 @@ export default {
 
 			// A very simple test query
 			let result = await client.query({ text: 'SELECT * FROM pg_tables' });
-
+			ctx.waitUntil(client.end());
```
Comment (rozenmd) [inline content/hyperdrive/learning/connect-to-postgres.md]: ditto here @elithrar

File: content/hyperdrive/_partials/_create-hyperdrive-config.md (hunk 1 above)
Comment (elithrar) [reply content/hyperdrive/_partials/_create-hyperdrive-config.md]: We do not require client.end

**Pull request merged**